ArduCopter/mode_throw.cpp

```diff
@@ -19,6 +19,9 @@ bool ModeThrow::init(bool ignore_checks)
     stage = Throw_Disarmed;
     nextmode_attempted = false;
 
+    // initialise the z controller
```

Comment:
It does no harm (besides consuming a bit of flash) but I don't think we need these upper four calls to init_z_controller().  The position controller is only run in Throw_HgtStabilise and Throw_PosHold and line 73 already has the init_z_controller_stopping_point().

Reply:
Removed them.